src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -152,6 +154,14 @@ def upload_signed_url(self, data, signed_url):
     """Uploads |data| to |signed_url|."""
     raise NotImplementedError
 
+  def sign_delete_url(self, remote_path, minutes=SIGNED_URL_EXPIRATION_MINUTES):
+    """Signs a DELETE URL for a remote file."""
+    raise NotImplementedError
+
+  def delete_signed_url(self, signed_url):
```

Comment:
rename to `delete_resource` ?

Reply:
Ehh I think current message makes sense.